src/client/app/reducers/github.ts

```diff
@@ -48,7 +48,8 @@ export function reducer(state = initialState, action: GitHubActions): GitHubStat
         }
 
         case GitHubActionTypes.SHARE_PRIVATE_GIST:
-        case GitHubActionTypes.SHARE_PUBLIC_GIST: {
+        case GitHubActionTypes.SHARE_PUBLIC_GIST:
+        case GitHubActionTypes.UPDATE_GIST: {
             AI.trackEvent(action.type);
             return { ...state, sharing: true };
```

Comment:
Added under UPDATE_INFO in 'snippet.ts' in 'reducers'; it logs the gist id using the same sha hash function.